sycl/plugins/hip/pi_hip.cpp

```diff
@@ -2936,11 +2936,14 @@ pi_result hip_piextProgramGetNativeHandle(pi_program program,
 ///
 /// \param[in] nativeHandle The native handle to create PI program object from.
 /// \param[in] context The PI context of the program.
+/// \param[in] ownNativeHandle tells if should assume the ownership of
+///            the native handle.
```

Comment:
```suggestion
/// \param[in] ownNativeHandle Indicates whether the created PI object should
///            take ownership of the native handle.
```
Just a little more specific.

Reply:
OK, I updated the wording